aries-site/src/data/mockData/users.json

```diff
@@ -0,0 +1,67 @@
+ [
```

Comment:
this doesn't like like proper json structure. I'd expect:
```
{
   "users": [
     ...
    ]
}
```

Reply:
okay I can change I just did similar from another file in here their all kinda different

Reply:
done